examples/client.go

```diff
@@ -603,23 +558,21 @@ func main() {
 					fmt.Printf("Host %s did not transtion into desired mode(s)\n", host)
 				}
 			}
-
-			fmt.Printf("error: %+v\n", err.Error())
-			os.Exit(1)
+			log.Fatalf("error: %+v\n", err.Error())
 		}
 
 		fmt.Print(result.String())
+
 	case "endMaintenance":
 		fmt.Println("Setting hosts to ACTIVE")
 		if hostList == "" {
-			fmt.Println("No hosts specified to drain")
-			os.Exit(1)
+			log.Fatalln("No hosts specified to drain")
```

Comment:
actually when using `log.Fatal*` you don't need to add `\n`, because they all went to the `Output` and it has the following https://golang.org/src/log/log.go?s=5252:5306#L139 so just `log.Fatalf("foo")` or `log.Fatal` the `\n` will be added automatically

````go
func (l *Logger) Output(calldepth int, s string) error {
	now := time.Now() // get this early.
	l.mu.Lock()
	defer l.mu.Unlock()
	// ... get file line using runtime
	l.buf = l.buf[:0]
	l.formatHeader(&l.buf, now, file, line)
	l.buf = append(l.buf, s...)
	if len(s) == 0 || s[len(s)-1] != '\n' {
		l.buf = append(l.buf, '\n')
	}
	_, err := l.out.Write(l.buf)
	return err
}
`````

Reply:
Awesome, I didn't know this. Kind of unexpected behavior, so thanks for pointing it out!